HelloMixpanel/HelloMixpanel/ViewController.m

```diff
@@ -159,7 +153,7 @@ - (IBAction)changeColor
     FCColorPickerViewController *colorPicker = [[FCColorPickerViewController alloc]
                                                 initWithNibName:@"FCColorPickerViewController"
                                                 bundle:[NSBundle mainBundle]];
-    colorPicker.color = [UIColor mp_applicationPrimaryColor];
+    colorPicker.color = [UIColor redColor];
```

Comment:
Yes, this header is longer exposed. This only sets the initial color of the picker.